align/pnr/checkers.py

```diff
@@ -48,13 +53,17 @@ def add_terminal( netName, layer, b, tag=None):
         terminals.append( { "netName": netName, "layer": layer, "rect": r})
 
         def f( gen, value, tag=None):
-            # value is in 2x units
+            # value is in units of 2nm
```

Comment:
I think you want to say `0.5nm`

Reply:
Thanks for the correction. I will change it.